src/client/pages/Landing.tsx

```diff
@@ -126,17 +135,143 @@ export const Landing: FC = () => {
     <Flex direction="column" height="100vh" bgColor="neutral.100">
       <Container maxW="80vw" bgColor="neutral.100">
         <LandingNavbar />
-        <HStack bgColor="neutral.100" pb="200px">
+        <Stack direction="row" bgColor="neutral.100" pb="200px">
           <VStack pt="20vh" alignItems="left">
             <Box textStyle="hero">Don't Know?</Box>
             <Box textStyle="hero">CheckFirst.</Box>
-            <Button mt="5vh !important" colorScheme="primary" width="120px">
+            <Button
+              onClick={learnMore}
+              mt="5vh !important"
+              colorScheme="primary"
+              width="120px"
+            >
               Learn More
             </Button>
           </VStack>
           <HStack h="100%" flex={1} justifyContent="center" />
-        </HStack>
+        </Stack>
       </Container>
+      <Container id="learn-more" maxW="100vw" pt="8vh" bgColor="neutral.50">
```

Comment:
```suggestion
      <Container id="learn-more" maxW="100vw" py="8vh" bgColor="neutral.50">
```

Based on the design, we need padding at the bottom too. 

![image](https://user-images.githubusercontent.com/3666479/109117960-a3f08b80-777d-11eb-9240-c97c45e9669f.png)